libs/src/sdk/api/tracks/TracksApi.ts

```diff
@@ -249,6 +252,59 @@ export class TracksApi extends TracksApiWithoutStream {
     return txReceipt
   }
 
+  /**
+   * Favorite a track
+   */
+  async saveTrack(
+    requestParameters: SaveTrackRequest,
+    writeOptions?: WriteOptions
+  ) {
+    // Parse inputs
+    const { userId, trackId, metadata } = parseRequestParameters(
+      'saveTrack',
+      SaveTrackSchema
+    )(requestParameters)
+
+    const response = await this.entityManager.manageEntity({
+      userId,
+      entityType: EntityType.TRACK,
+      entityId: trackId,
+      action: Action.SAVE,
+      metadata: metadata && JSON.stringify(metadata),
+      auth: this.auth,
+      ...writeOptions
+    })
+    const txReceipt = response.txReceipt
```

Comment:
[No action required since not in the scope of this PR but] wonder if we should return something else for successful write calls, since idt the txReceipt is helpful and people prob won't know what it is

Reply:
Yes! I have an item in the write sdk sync agenda to discuss. But also Alec has put some thought into how we could return the actual result value or error https://www.notion.so/audiusproject/WIP-Roundtrip-Write-SDK-Brainstorm-fbc9dcd4d90d4d70b5f331c9076d60e6